worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -423,7 +427,9 @@ class PushDataHandler extends BaseMessageHandler with Logging {
         } else {
           StatusCode.PUSH_DATA_FAIL_SLAVE.getMessage()
         }
-      callback.onFailure(new Exception(message, exception))
+      callback.onFailure(new Exception(
+        s"$message! ${locations.map(_.toString).mkString("{", ", ", "}")}",
```

Comment:
If the purpose is to log the worker location, then it's unnecessary to print out all locations here, since they all belong the the same worker. Maybe locations.head is enough.